### ISSUE TITLE
DynamincSelectModel prefix

### ISSUE THREAD

**Issue by stemyke** [udos86/ng2-dynamic-forms#290]
I can't add prefix to Dynamic select model it should be possible

**Comment by udos86** [comment]
@stemyke Ok, I'll have to check how this supported in Bootstrap and Foundation.

**Comment by udos86** [comment]
Closed as this is not supported by Bootstrap or Foundation. Use `<template>` instead.

**Comment by stemyke** [comment]
I don't know how Foundation works, but I managed to do that in Bootstrap just by copying this two file to my src:

@ng2-dynamic-forms\ui-bootstrap\src\dynamic-form-bootstrap.component.html
@ng2-dynamic-forms\ui-bootstrap\src\dynamic-form-bootstrap.component.ts

and put the select field into a container div like you did with the normal input field.

I also copied 

@ng2-dynamic-forms\core\src\model\select\dynamic-select.model.ts

and added, this two lines:

    @serializable() prefix: string | null;
    @serializable() suffix: string | null;

to DynamicSelectModelConfig<T> and this two lines:

        this.prefix = getValue(config, "prefix", null);
        this.suffix = getValue(config, "suffix", null);

to DynamicSelectModel<T> constructor.

With this I have a nice icon (now it is wrong) before the Country select field, but it is not a nice solution to copy your code to my project.

![image](https://cloud.githubusercontent.com/assets/9153908/23879923/838b5e38-0850-11e7-83f2-eeefc02af8cb.png)

I can't do the same thing with template, because with it, I can't position my prefix icons to the same line.
